### ISSUE TITLE
Using setCookiesPath creates 2 cookies

### ISSUE THREAD

**Issue by parruc** [piwik/piwik#8112]
I have some subdirs that I want to track as different sites. Following the docs I called, in the tracking code, setCookiesPath but two id cookies are created the first on / and the second on the right subdir.

I've debugged the js and the first cookie is created at constructor time 

rows: 5340 and 5341
> var applyFirst  = ['disableCookies', 'setTrackerUrl', 'setAPIUrl', 'setUserId', 'setSiteId', 'enableLinkTracking'];
 > _paq = applyMethodsInOrder(_paq, applyFirst);

In particular setSiteId sets the id cookie too soon (before I can call setCookiesPath) my and I end up having 2 id cookies, one for the root and one for my selected subdir.

**Comment by mattab** [comment]
Hi @parruc 

could you paste here your Piwik JavaScript Tracking code?

**Comment by parruc** [comment]
Here it is the tracking code I'm using @mattab:

```html
<!-- Piwik -->
<script type="text/javascript">
     var paq = paq || [];
     _paq.push(['setCookiePath', '/people/john.doe']);
     _paq.push(['trackPageView']);
     _paq.push(['enableLinkTracking']);
    (function () {
        var u = "https://example.com/";
        _paq.push(['setTrackerUrl', u + 'piwik.php']);
        _paq.push(['setSiteId', 14302]);
        var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
        g.type = 'text/javascript'; g.async = true; g.defer = true; g.src = u + 'piwik.js'; s.parentNode.insertBefore(g, s);
  })();
</script>
<noscript><p><img src="https://example.com/piwik.php?idsite=14302" style="border:0;" alt="" /></p></noscript>
<!-- End Piwik Code -->
```

**Comment by parruc** [comment]
Was you able to reproduce it?

**Comment by mattab** [comment]
Fixed in #8265  @parruc if it's not fixed please re-open
